remove use of Guava @VisibleForTesting on all package-private members

msgilligan opened the pull request:
Both methods it is used on are already package-private and one is deprecated.

schildbach commented:
If we replace `@VisibleForTesting` with just a comment, imho we should do that for the entire codebase at once.

msgilligan commented:
@schildbach I implemented (2) above ("Replace all usages of  @VisibleForTesting on package-private items with a comment"), rebased, force-pushed, and updated the title.

schildbach commented:
Yes, that's a good start. Merged.
